# Q1) A F&B manager wants to determine whether there is any significant difference in the diameter of the cutlet between two units. A randomly selected sample of cutlets was collected from both units and measured? Analyze the data and draw inferences at 5% significance level. Please state the assumptions and tests that you carried out to check validity of the assumptions.

In [2]:
import pandas as pd
import numpy as np
from scipy import stats
import matplotlib as plt
import seaborn as sn

In [3]:
data = pd.read_csv("C:\\Users\\Anks\\Downloads\\Cutlets (1).csv")


In [6]:
data.head(5)

Unit A  Unit B
0  6.8090  6.7703
1  6.4376  7.5093
2  6.9157  6.7300
3  7.3012  6.7878
4  7.4488  7.1522

In [7]:
Unit_A = data.iloc[:,0]
Unit_B = data.iloc[:,1]


In [8]:
stat_value,p_value = stats.ttest_ind(Unit_A,Unit_B)
print(stat_value,p_value)

0.7228688704678063 0.4722394724599501


Ans) As p-value(0.4722) is greater than alpha significance(0.05) we can conclude that - "We fail to reject null hypothesis i.e. their is no difference in the diameters of the cutlets."

# Q2) A hospital wants to determine whether there is any difference in the average Turn Around Time (TAT) of reports of the laboratories on their preferred list. They collected a random sample and recorded TAT for reports of 4 laboratories. TAT is defined as sample collected to report dispatch.Analyze the data and determine whether there is any difference in average TAT among the different laboratories at 5% significance level.

In [12]:
data1=pd.read_csv("C:\\Users\\Anks\\Downloads\\LabTAT.csv")

In [13]:
data1

Laboratory 1  Laboratory 2  Laboratory 3  Laboratory 4
0          185.35        165.53        176.70        166.13
1          170.49        185.91        198.45        160.79
2          192.77        194.92        201.23        185.18
3          177.33        183.00        199.61        176.42
4          193.41        169.57        204.63        152.60
..            ...           ...           ...           ...
115        178.49        170.66        193.80        172.68
116        176.08        183.98        215.25        177.64
117        202.48        174.54        203.99        170.27
118        182.40        197.18        194.52        150.87
119        182.09        215.17        221.49        162.21

[120 rows x 4 columns]

In [14]:
lab1 = data.iloc[:,0]
lab2 = data.iloc[:,1]
lab3 = data.iloc[:,2]
lab4 = data.iloc[:,3]


In [16]:
stat_value,p_value = stats.f_oneway(lab1,lab2,lab3,lab4)
print(stat_value,p_value)



118.70421654401437 2.1156708949992414e-57


Ans) As p-value(2.1157) is greater than alpha significance(0.05) we can conclude that - "We accept alternate hypothesis i.e. their is no significant inter-relation between all the laboratories"